packages/@react-aria/overlays/src/useModal.tsx

```diff
@@ -106,16 +106,28 @@ export function OverlayProvider(props: ModalProviderProps) {
   );
 }
 
+interface OverlayContainerProps extends ModalProviderProps {
+  containerRootElement?: HTMLElement
```

Comment:
Maybe `portalContainer` as suggested in #1574 would be more clear what this is for?

Reply:
sounds reasonable